### PR TITLE
chore(aalborg_theme): Returns caret to avatars on hover

### LINKED ISSUE #8629: master: Hover menu icon doesn't work
Noticed it in theme sandbox. It looks correctly set to use fa-caret-down, but CSS isn't right.

### PR DESCRIPTION
**Pull request by mrclay**
BREAKING CHANGE:
The file mod/aalborg_theme/graphics/avatar_menu_arrows.png has been removed.

Fixes #8629

### CODE REVIEW

**Comment by jdalsem** [comment]
this change in Aalborg was intentionally done for touch i believe... this now introduces the "default" elgg problem that you have problems reaching the dropdown menu with touch devices...

**Comment by jdalsem** [comment]
otherwise LGTM

**Comment by mrclay** [comment]
I certainly don't want to interfere with touch. @PerJensen got a recommendation here?

**Comment by PerJensen** [comment]
The graphics were added to make it easier to hit on touch screens. It is just as large as the icon profile, 40 x 40, as opposed to the default which is 16 x 16.

**Comment by mrclay** [comment]
@jdalsem Have you tested this? What particularly in my changes would break the existing touch-to-open user menus?

File: mod/aalborg_theme/views/default/aalborg_theme/css.php [hunk 1]
```diff
@@ -73,14 +73,15 @@
...
 	width: 100%;
 	height: 100%;
 }
 .elgg-icon-hover-menu-hover:before,
 .elgg-icon-hover-menu:before {
-	content: none;
+	position: absolute;
+	bottom: 0;
+	right: 0;
+	color: #fff;
```
Comment (jdalsem) [inline mod/aalborg_theme/views/default/aalborg_theme/css.php]: do not change to white as that cause visual issues... just leave as default (or switch to aalborgs BLACK color, as used in the image)

**Comment by jdalsem** [comment]
@mrclay tested it, and this approach has no touch issues. If you could respond to the color comment and also remove the image from the aalborgtheme this is good to merge

**Pull request merged**